src/rules/property-no-unknown/README.md

```diff
@@ -85,3 +87,20 @@ The following patterns are *not* considered warnings:
   custom: 10px;
 }
 ```
+### `checkPrefixed: true | false` (default: `false`)
+
+This option forces to check vendor prefixed properties.
```

Comment:
> If `true`, this rule will check vendor-prefixed properties.

SGTM.

> @jeddy3 any thoughts on this?

Tricky. Kind of makes me wonder why we are ignoring them by default. Perhaps we shouldn't and just patch `at-rule-no-unknown` so that it doesn't ignore them as, I believe, that's the only `*-no-unknown` rule that does that at the moment? 

Then change this option to `ignorePrefixed` and then say for "For compatibility with csslint that you'd want to turn the option on".